src/main/java/net/sf/jabref/logic/formatter/bibtexfields/HtmlToUnicodeFormatter.java

```diff
@@ -0,0 +1,36 @@
+package net.sf.jabref.logic.formatter.bibtexfields;
+
+import net.sf.jabref.logic.formatter.Formatter;
+import net.sf.jabref.logic.l10n.Localization;
+import net.sf.jabref.logic.layout.LayoutFormatter;
+
+import org.apache.commons.lang3.StringEscapeUtils;
+
+public class HtmlToUnicodeFormatter implements LayoutFormatter, Formatter {
```

Comment:
Refs: #8 None of the current formatters are documented and I won't build documentation for all as part of this PR. This has to go separately.